appveyor.yml

```diff
@@ -22,7 +22,7 @@ test_script:
 after_test:
   - if defined APPVEYOR_REPO_TAG_NAME (
       npm run rebuild &&
-      7z a -ttar %APPVEYOR_REPO_TAG_NAME%-win-%PLATFORM%.tar node_libui.node &&
+      7z a -ttar %APPVEYOR_REPO_TAG_NAME%-win-%PLATFORM%.tar libui.node &&
```

Comment:
Unfortunately, on windows the rename does not work: macOS & linux strip 'lib' from the file name if it should normally begin with 'lib'. Eg. libui.node -> ui.node. Windows does not.